Mathlib/Data/Nat/Digits.lean

```diff
@@ -738,6 +742,81 @@ theorem eleven_dvd_of_palindrome (p : (digits 10 n).Palindrome) (h : Even (digit
   exact eq_zero_of_neg_eq this.symm
 #align nat.eleven_dvd_of_palindrome Nat.eleven_dvd_of_palindrome
 
+/-! ### `Nat.toDigits` length -/
+
+lemma to_digits_core_lens_eq_aux (b f : Nat) :
```

Comment:
While we're at it, let's fix the names.
```suggestion
lemma toDigitsCore_length_eq_aux (b f : Nat) :
```

Reply:
I promise I will, but I split this out from  #9627 specifically to avoid making changes while moving code.